operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -248,41 +251,42 @@ func (r *ReconcileIstioOperator) Reconcile(request reconcile.Request) (reconcile
 		err := r.client.Update(context.TODO(), iop)
 		if err != nil {
 			if errors.IsNotFound(err) {
-				log.Infof("Could not add finalizer to %v: the object was deleted", request)
+				scope.Infof("Could not add finalizer to %s: the object was deleted.", iopName)
 				return reconcile.Result{}, nil
 			} else if errors.IsConflict(err) {
-				log.Infof("Could not add finalizer to %v due to conflict. Operation will be retried in next reconcile attempt", request)
+				scope.Infof("Could not add finalizer to %s due to conflict. Operation will be retried in next reconcile attempt.", iopName)
 				return reconcile.Result{}, nil
 			}
-			log.Errorf("Failed to update IstioOperator with finalizer, %v", err)
+			scope.Errorf(errdict.OperatorFailedToAddFinalizer, "Failed to add finalizer to IstioOperator CR %s: %s", iopName, err)
 			return reconcile.Result{}, err
 		}
 	}
 
-	log.Info("Updating IstioOperator")
+	scope.Info("Updating IstioOperator")
 	var err error
 	iopMerged := &iopv1alpha1.IstioOperator{}
 	*iopMerged = *iop
 	iopMerged.Spec, err = mergeIOPSWithProfile(iopMerged)
 
 	if err != nil {
-		log.Errorf("failed to generate IstioOperator spec, %v", err)
+		scope.Errorf(errdict.OperatorFailedToMergeUserIOP, "failed to merge base profile with user IstioOperator CR %s, %s", iopName, err)
 		return reconcile.Result{}, err
 	}
 
 	if _, ok := iopMerged.Spec.Values["global"]; !ok {
 		iopMerged.Spec.Values["global"] = make(map[string]interface{})
 	}
 	globalValues := iopMerged.Spec.Values["global"].(map[string]interface{})
-	log.Info("Detecting third-party JWT support")
+	scope.Info("Detecting third-party JWT support")
 	var jwtPolicy util.JWTPolicy
 	if jwtPolicy, err = util.DetectSupportedJWTPolicy(r.config); err != nil {
-		log.Warnf("Failed to detect third-party JWT support: %v", err)
+		// TODO: add to dictionary. {status=New, assignee=}
```

Comment:
What is `{status=New, assignee=}`?

Reply:
assignee=howardjohn :-)
i want to have some annotation to keep track of log entries that need fixing but also want to capture WontFix type of resolution if the assignee things the status quo is fine. how about this instead:
```
TODO(): add to dictionary. When resolved, replace this sentence with Done or WontFix.
```